wombat/src/main/cpp/subsystems/Arm.cpp

```diff
@@ -71,10 +79,13 @@ void Arm::OnUpdate(units::second_t dt) {
   voltage *= armLimit;
 
   // units::newton_meter_t torqueLimit = 10_kg * 1.4_m * 6_mps_sq;
-  // units::volt_t voltageMax = _config.leftGearbox.motor.Voltage(torqueLimit, _config.leftGearbox.encoder->GetEncoderAngularVelocity());
-  // units::volt_t voltageMin = _config.leftGearbox.motor.Voltage(-torqueLimit, _config.leftGearbox.encoder->GetEncoderAngularVelocity());
+  // units::volt_t voltageMax = _config.leftGearbox.motor.Voltage(torqueLimit,
+  // _config.leftGearbox.encoder->GetEncoderAngularVelocity()); units::volt_t
+  // voltageMin = _config.leftGearbox.motor.Voltage(-torqueLimit,
+  // _config.leftGearbox.encoder->GetEncoderAngularVelocity());
 
-  // voltage = units::math::max(units::math::min(voltage, voltageMax), voltageMin);
+  // voltage = units::math::max(units::math::min(voltage, voltageMax),
+  // voltageMin);
   units::volt_t voltageMin = -5.5_V;
```

Comment:
i know this was definitely my fault but min voltage should probably be 0 since we use DC power

Reply:
:+1: didnt edit it will change it